tpm2/Makefile

```diff
@@ -254,91 +255,124 @@ INDENT_OPTS += -l80#                     max non-comment line length is 80
 #====================== START: RULES =========================================
 
 .PHONY: all
-all: pre kmyth-lib kmyth-seal kmyth-unseal kmyth-getkey
+all: pre clean-backups logger-lib tpm-util-lib \
+     $(BIN_DIR)/kmyth-seal \
+     $(BIN_DIR)/kmyth-unseal \
+     $(BIN_DIR)/kmyth-getkey
 
-kmyth-lib: $(LOGGER_OBJECTS) \
-           $(UTIL_OBJECTS) \
-           $(CIPHER_OBJECTS) \
-           $(TPM_OBJECTS)
+.PHONY: pre
+pre:
+	indent $(INDENT_OPTS) $(SOURCE_FILES)
+	indent $(INDENT_OPTS) $(HEADER_FILES)
+
+.PHONY: logger-lib
```

Comment:
Do this target really need to be phony?

Reply:
So, I made $(LIB_DIR)/libkmyth-logger.so a non-phony target, but I liked having a simpler make target name, so I kept a make logger-lib phony target that simply has $(LIB_DIR)/libkmyth-logger.so as a pre-requisite.